src/src/AdminBundle/Command/CreateSuperuserCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace AdminBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use AdminBundle\Entity\Admin;
+
+class CreateSuperuserCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName('admin:create-superuser')
+             ->setDescription('Creates a new Admin user')
+             ->setHelp('Create a new Administrator. Use it to log in to the admin website.')
+             ->addArgument('email', InputArgument::REQUIRED, 'E-mail of the Administrator.')
+             ->addArgument('password', InputArgument::REQUIRED, 'Password of the Administrator.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $newAdmin = new Admin($input->getArgument('email'));
+        $encoder = $this->getContainer()->get('security.encoder_factory')->getEncoder($newAdmin);
+        $newAdmin->setPassword($encoder->encodePassword($input->getArgument('password'), $newAdmin->getSalt()));
```

Comment:
https://github.com/TIS2017/Nobelisti/blob/108c4ef8bfb84af994cd9a0ffda26f3b9e226eee/src/src/AdminBundle/Controller/AdminController.php#L30 tu sa to robi inak. je to jedno alebo je jedna z tychto veci zla?

Reply:
Podla toho co som pochopil je to jedno, to co robim ja je asi len krok navyse. Mozem to prerobit